src/components/InventoryDetail/ApplicationDetails.js

```diff
@@ -105,7 +106,7 @@ const ApplicationDetails = ({
         </Tabs>
       </section>
       <section>
-        {activeTabs?.length &&
+        {activeTabs?.length > 0 &&
```

Comment:
We could wrap the whole `Fragment` in this and either render nothing or a `Spinner`, while it has no `entity` and no `activeTabs`.

Reply:
Its small, but I like this change. I've gotten a few messages to slap this on qa already so im going to merge and then make a separate PR for this.

Reply:
@adonispuente Oh, when you do that. a minor thing we could also improve... the `React.Fragment` is redundant. All child components are already wrapped in one `section`.